bubblewrap.c

```diff
@@ -1327,6 +1333,10 @@ main (int    argc,
   if (!is_privileged)
     opt_unshare_user = TRUE;
 
+  if (opt_unshare_user_try &&
+      stat ("/proc/self/ns/user", &sbuf) == 0)
```

Comment:
(However I was wrong about this not working on Debian initially, you're right it is OK there)